uuid/example_test.go

```diff
@@ -0,0 +1,43 @@
+// Run Tarantool before execution:
+// TEST_TNT_WORK_DIR=/tmp TEST_TNT_LISTEN=3013 tarantool uuid/config.lua
+
+package uuid_test
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/tarantool/go-tarantool"
+	_ "github.com/tarantool/go-tarantool/uuid"
+)
+
+func Example() {
+	server := "127.0.0.1:3013"
+	opts := tarantool.Opts{
+		Timeout:       500 * time.Millisecond,
+		Reconnect:     1 * time.Second,
+		MaxReconnects: 3,
+		User:          "test",
+		Pass:          "test",
+	}
+	client, err := tarantool.Connect(server, opts)
+	if err != nil {
+		log.Fatalf("Failed to connect: %s", err.Error())
+	}
+
+	spaceNo := uint32(524)
+
+	id, uuidErr := uuid.Parse("c8f0fa1f-da29-438c-a040-393f1126ad39")
+	if uuidErr != nil {
+		log.Fatalf("Failed to prepare uuid: %s", uuidErr)
+	}
+
+	resp, err := client.Replace(spaceNo, []interface{}{id})
+
+	fmt.Println("UUID tuple replace")
+	fmt.Println("Error", err)
+	fmt.Println("Code", resp.Code)
+	fmt.Println("Data", resp.Data)
```

Comment:
https://pkg.go.dev/testing#hdr-Examples:
> The package also runs and verifies example code. Example functions may include a concluding line comment that begins with "Output:" and is compared with the standard output of the function when the tests are run. (The comparison ignores leading and trailing space.) These are examples of an example:

Added output to all examples except custom unpacking.

Reply:
There is no output in subpackage examples

Reply:
Right. This example compiled and not executed because I don't know how to make it executable for certain Tarantool versions. Otherwise example with fail on Tarantool versions without UUID support.

Reply:
Oh, yeah, I forgot about this one. Let's skip it then